File: src/main/java/com/example/examplemod/models/TelemetryNode.java
package com.example.examplemod.models;

import java.util.List;

public class TelemetryNode {
    public String TelemetryNodeId;
    public List<MinimizedBlockPos> LinkedMachines;

    public TelemetryNode(String telemetryNodeId, List<MinimizedBlockPos> linkedMachines) {
        this.TelemetryNodeId = telemetryNodeId;
        this.LinkedMachines = linkedMachines;
    }
}

File: JazzyFactoryTelemetryApi/Models/TelemetryNode.cs
namespace JazzyFactoryTelemetryApi.Models;
using System.Collections.Generic;
using JazzyFactoryTelemetryApi.Models;

public class TelemetryNode
{
    public string TelemetryNodeId { get; set; }
    public List<MinimizedBlockPos> LinkedMachines { get; set; }
}